src/pudl/metadata/enums.py

```diff
@@ -73,6 +73,111 @@
 See https://www.eia.gov/electricity/data/eia411/#tabs_NERC-3.
 """
 
+EIA_REGIONS: list[str] = [
+    "CAL",
+    "CAN",
+    "CAR",
+    "CENT",
+    "FLA",
+    "MEX",
+    "MIDA",
+    "MIDW",
+    "NE",
+    "NW",
+    "NY",
+    "SE",
+    "SW",
+    "TEN",
+    "TEX",
+]
+
+EIA_SUBREGIONS: list[str] = [
```

Comment:
sigh why are these (string) numbers and letters?

Reply:
Because that's what they reported. The field has to be a string because most of them are strings, but I think these subregion IDs are just whatever tf the ISO felt like using.